build-tools/package.json

```diff
@@ -73,9 +73,9 @@
 		"@commitlint/cli": "^17.6.6",
 		"@commitlint/config-conventional": "^17.6.6",
 		"@commitlint/cz-commitlint": "^17.5.0",
-		"@fluid-tools/build-cli": "~0.40.0",
+		"@fluid-tools/build-cli": "~0.44.0",
```

Comment:
Unfortunately, due to quirks in build-tools you need to also manually update references like this in some build-tools packages:

```json
		"@fluidframework/build-tools-bin": "npm:@fluidframework/build-tools@~0.40.0",
```